b2sdk/_test_manager/api.py

```diff
@@ -0,0 +1,188 @@
+                        NO_RETENTION_FILE_SETTING, True
+                    )
+                elif file_version_info.file_retention.mode == RetentionMode.COMPLIANCE:
+                    if file_version_info.file_retention.retain_until > current_time_millis():  # yapf: disable
+                        print(
+                            'File version: %s cannot be removed due to compliance mode retention' %
+                            (file_version_info.id_,)
+                        )
+                        files_leftover = True
+                        continue
+                elif file_version_info.file_retention.mode == RetentionMode.NONE:
+                    pass
+                else:
+                    raise ValueError(
+                        'Unknown retention mode: %s' % (file_version_info.file_retention.mode,)
+                    )
+            if file_version_info.legal_hold.is_on():
+                print('Removing legal hold from file version:', file_version_info.id_)
+                self.update_file_legal_hold(
+                    file_version_info.id_, file_version_info.file_name, LegalHold.OFF
+                )
+            print('Removing file version:', file_version_info.id_)
+            try:
+                self.delete_file_version(file_version_info.id_, file_version_info.file_name)
+            except FileNotPresent:
+                print(
+                    'It seems that file version %s has already been removed' %
+                    (file_version_info.id_,)
+                )
+
+        if files_leftover:
+            print('Unable to remove bucket because some retained files remain')
+        else:
+            print('Removing bucket:', bucket.name)
+            try:
+                self.delete_bucket(bucket)
+            except BucketIdNotFound:
+                print('It seems that bucket %s has already been removed' % (bucket.name,))
+        print()
+
+    def clean_buckets(self) -> None:
+        for bucket in self.buckets:
+            with contextlib.suppress(BucketIdNotFound, v3BucketIdNotFound, NonExistentBucket):
+                self.clean_bucket(bucket)
+        self.buckets = []
+
+    def clean_all_buckets(self) -> None:
+        buckets = self.list_buckets()
+        print(f'Total bucket count: {len(buckets)}')
+
+        for bucket in buckets:
+            if not bucket.name.startswith(BUCKET_NAME_PREFIX):
+                print(f'Skipping bucket removal: "{bucket.name}"')
+                continue
+
+            print(f'Removing bucket: "{bucket.name}"')
+            try:
+                self.clean_bucket(bucket)
+            except (BucketIdNotFound, v3BucketIdNotFound):
+                print(f'It seems that bucket "{bucket.name}" has already been removed')
+
+        buckets = self.list_buckets()
+        print(f'Total bucket count after cleanup: {len(buckets)}')
+        for bucket in buckets:
+            print(bucket)
+
+    def count_and_print_buckets(self) -> int:
```

Comment:
A simple search provides me with no usages of this function. Can you point me to an actual usage, please?

Reply:
It was intended to be a bit of statistic in total as you write in the comment below also we can put this function into `cleanup_buckets` function to always track how many buckets have been created when closing tests